user-operator/src/main/java/io/strimzi/operator/user/operator/KafkaUserOperator.java

```diff
@@ -106,6 +109,41 @@ public void stop() {
         scramCredentialsOperator.stop();
     }
 
+    /**
+     * Creates the informer for given resource type to inform on all instances in
+     * given namespace (or cluster-wide) matching the selector. The informer
+     * returned by this method is not running and has to be started by the code
+     * using it.
+     *
+     * @param namespace        Namespace on which to inform
+     * @param selectorLabels   Selector which should be matched by the resources
+     * @param resyncIntervalMs The interval in which the resync of the informer
+     *                         should happen in milliseconds
+     *
+     * @return Informer instance
+     *
+     * @see io.strimzi.operator.common.operator.resource.concurrent.AbstractNamespacedResourceOperator#informer(String,
+     *      Map, long)
+     */
+    public SharedIndexInformer<KafkaUser> informer(String namespace, Map<String, String> selectorLabels, long resyncIntervalMs) {
```

Comment:
An alternative to this and the following proxy method would be to have `KafkaUserOperator` extend `CrdOperator`.